runtime/near-vm-runner/src/cache.rs

```diff
@@ -47,11 +36,10 @@ fn get_key(code_hash: &[u8], code: &[u8], vm_kind: VMKind, config: &VMConfig) ->
         // Sometimes caller doesn't compute code_hash, so hash the code ourselves.
         Err(_e) => near_primitives::hash::hash(code),
     };
-    let key = ContractCacheKey::Version2 {
+    let key = ContractCacheKey::Version1 {
```

Comment:
we can keep this as version1, and hide version2 so node is exactly behaves as before. Then make some benches to see whether we should adopt version 2 for some or all contracts.

Reply:
Revert is safer. Once we have proof that this change is needed for performance gains we can proceed to put it. Otherwise any new change should be guarded by a compilation feature in case it has a chance to affect the node performance.